docs/main/introduction-to-openebs/features.mdx

```diff
@@ -65,7 +65,7 @@ OpenEBS Features, like any storage solution, can be broadly classified into the
 
 <TwoColumn left="1fr" right="200px">
   <p>
-    The backup and restore of OpenEBS volumes works with Kubernetes backup and restore solutions such as Velero (formerly Heptio Ark) via open source OpenEBS Velero-plugins. Data backup to object storage targets such as AWS S3, GCP Object Storage or MinIO are frequently deployed using the OpenEBS incremental snapshot capability. This storage level snapshot and backup saves a significant amount of bandwidth and storage space as only incremental data is used for backup.
+    The backup and restore of OpenEBS volumes works with Kubernetes backup and restore solutions such as Velero (f.k.a Heptio Ark) via open source OpenEBS Velero-plugins. Data backup to object storage targets such as AWS S3, GCP Object Storage or MinIO are frequently deployed using the OpenEBS incremental snapshot capability. This storage level snapshot and backup saves a significant amount of bandwidth and storage space as only incremental data is used for backup.
```

Comment:
we can remove f.k.a Heptio Ark, that's very long time ago and now, Velero name has established it's popularity